images/custom-scorecard-tests/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Build the custom-scorecard-tests binary
+FROM --platform=x86_64 golang:1.17.9 as builder
```

Comment:
Can we use the same images as in `/ci` directory? Unification could be a good thing here.

Reply:
+1